config.TEMPLATE.inc.php

```diff
@@ -317,6 +317,9 @@
 ; smtp_username = username
 ; smtp_password = password
 
+; Enable suppressing verification of SMTP certificate in PHPMailer
```

Comment:
I'd suggest adding a note that this is not recommended as per the PHPMailer documentation (see under "Updating CA certificates" at https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting).